AUTHORS

```diff
@@ -1,43 +1,55 @@
 The following people, in alphabetical order, have either authored or signed
 off on commits in the Cilium repository:
 
-Alban Crequy          alban.crequy@gmail.com
-Alexander Alemayhu    alexander@alemayhu.com
-Alexei Starovoitov    ast@kernel.org
-Amre Shakimov         amre@covalent.io
-André Martins         andre@cilium.io
-Ashwin Paranjpe       ashwinp.work@gmail.com
-Craig Box             craig.box@gmail.com
-Cynthia Thomas        cynthia@covalent.io
-Dan Wendlandt         danwent@gmail.com
-Daniel Borkmann       daniel@cilium.io
-Daniel Dao            dqminh89@gmail.com
-Daniel (Shijun) Qian  sj.daniel@gmail.com
-Dom Del Nano          ddelnano@gmail.com
-Eloy Coto             eloy.coto@gmail.com
-Faiyaz Ahmed          faiyaza@gmail.com
-Gianluca Arbezzano    gianarb92@gmail.com
-Ian Vernon            ianvernon@covalent.io
-Ivar Lazzaro          ivarlazzaro@gmail.com
-Jan-Erik Rediger      badboy@archlinux.us
-Jarno Rajahalme       jarno@covalent.io
-John Fastabend        john@covalent.io
-Jon Davies            jpds@protonmail.com
-Katarzyna Borkmann    kasia@iogearbox.net
-Madhu Challa          madhu@cilium.io
-Marcin Skarbek        git@skarbek.name
-Michi Mutsuzaki       michi@covalent.io
-Peiqi Shi             uestc.shi@gmail.com
-Raghu Gyambavantha    r.grizzly@gmail.com
-Ross Guarino          rssguar@gmail.com
-Russell Bryant        russell@russellbryant.net
-Sergey Generalov      sergey@genbit.ru
-Steven Ceuppens       steven.ceuppens@icloud.com
-Thomas Bachman        tbachman@yahoo.com
-Thomas Graf           thomas@cilium.io
-Tobias Klauser        tklauser@distanz.ch
-Tony Lambiris         tony@criticalstack.com
-Trevor Roberts Jr     trevor.roberts.jr@gmail.com
+4ast <alexei.starovoitov@gmail.com>
```

Comment:
`s/4ast/Alexei Starovoitov/`?

Reply:
This is automatically generated now based on git commits.